.env

```diff
@@ -92,13 +92,13 @@ DEVTOOLSET_VERSION=
 # Used through docker-compose.yml and serves as the default version for the
 # ci/scripts/install_vcpkg.sh script. Prefer to use short SHAs to keep the
 # docker tags more readable.
-VCPKG="a42af01b72c28a8e1d7b48107b33e4f286a55ef6"    # 2023.11.20 Release
+VCPKG="a34c873a9717a888f58dc05268dea15592c2f0ff"    # 2024.03.25 Release
```

Comment:
We should not use this revision. Because this uses xz 5.6.0: https://github.com/microsoft/vcpkg/tree/2024.03.25/ports/liblzma

Reply:
Thanks @kou , so this is the newer we can use until vcpkg creates a new tag:
https://github.com/microsoft/vcpkg/tree/2024.02.14/ports/liblzma

Reply:
Yes. But it seems that we don't need this PR because https://github.com/tukaani-project/xz is enabled again.

Reply:
Indeed: https://github.com/microsoft/vcpkg/pull/37841#issuecomment-2045968110. That should mean it should just work again (the vcpkg version we are currently using is an older one that has non-affected version of liblzma)